modules/gapi/src/api/render_ocv.cpp

```diff
@@ -222,5 +225,73 @@ void drawPrimitivesOCVYUV(cv::Mat &in, const Prims &prims)
 
 } // namespace draw
 } // namespace wip
+
+void cv::gapi::ocv::createBitMask(const cv::gapi::wip::draw::Text& text_prim,
+                                  cv::Mat& mask,
+                                  int* baseline) {
+    GAPI_Assert(baseline);
+
+    cv::Size mask_size = cv::getTextSize(text_prim.text, text_prim.ff,
+                                         text_prim.fs, text_prim.thick,
+                                         baseline);
+    *baseline += text_prim.thick;
+
+    mask.create(mask_size.height + *baseline, mask_size.width, CV_8UC1);
+    mask = cv::Scalar::all(0);
+
+    cv::Point org((mask.cols - mask_size.width) / 2,
+                  (mask.rows + mask_size.height - *baseline) / 2);
+
+    cv::putText(mask, text_prim.text, org, text_prim.ff,
+                text_prim.fs, 255, text_prim.thick);
+}
+
+void cv::gapi::ocv::blendImage(const cv::Mat& img,
+                                     const cv::Mat& alpha,
+                                     const cv::Point& org,
+                                     cv::Mat background)
+{
+    GAPI_Assert(img.type()   == CV_32FC3);
```

Comment:
Are you sure it must be 32FC3? What's the problem with 8U?

Reply:
+1

Reply:
Fixed